src/styles.css

```
@@ -4,9 +4,9 @@ body {
 }
 
 .header {
-  height: 50px;
+  height: 49px;
```

Why couldn't you change the box-sizing instead?
```suggestion
  box-sizing: border-box;
  height: 50px;
```